charts/super-agent/templates/job-manifests.yaml

```diff
@@ -0,0 +1,34 @@
+{{- if .Values.helm.create -}}
+{{- $configMapName := include "newrelic.common.naming.truncateToDNSWithSuffix" ( dict "name" (include "newrelic.common.naming.fullname" .) "suffix" "job-manifests" ) -}}
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  annotations:
+    helm.sh/hook: pre-delete,post-install,post-upgrade
+    helm.sh/hook-delete-policy: before-hook-creation,hook-succeeded
+    helm.sh/hook-weight: "10000"
+  name: {{ $configMapName }}
+  namespace: {{ .Release.Namespace }}
+data:
+{{- /*
+Let's start with some background that allow to understand why configmap has manifests on it.
+
+Helm behaves differently depending on where you.
```

Comment:
"Helm behaves differently depending on where you."?

Reply:
You have seen nothing. And you have not seen the `push --force` I made. (Solved on [#2cd5d8c](https://github.com/newrelic/helm-charts/pull/1175/commits/2cd5d8cde4029707fdf1cccc58c411dae53f07a5))